From 1ae9ea8494000800f386d59d30b04f14d005f4e0 Mon Sep 17 00:00:00 2001
From: Yu Liao <liaoyu15@huawei.com>
Date: Thu, 13 Oct 2022 14:47:16 +0800
Subject: [PATCH 008/595] [Huawei] x86/ioapic: remove useless pr_debug in
 ioapic_sync_hardware_data

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187795

---------------------------

ioapic_sync_hardware_data wad added by commit 0763b481a1ca ([Huawei]
x86/ioapic: add an interface for driver to sync hardware data into memory),
pr_debug in this function caused some warnings, so remove it.

If it turns out to be need for debug, this commit can always be reverted.

Signed-off-by: Yu Liao <liaoyu15@huawei.com>
---
 arch/x86/kernel/apic/io_apic.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/x86/kernel/apic/io_apic.c b/arch/x86/kernel/apic/io_apic.c
index 34bc1e3325a8..18c006c95d3f 100644
--- a/arch/x86/kernel/apic/io_apic.c
+++ b/arch/x86/kernel/apic/io_apic.c
@@ -1220,8 +1220,6 @@ void ioapic_sync_hardware_data(int irq)
 		entry = ioapic_read_entry(pin_list->apic, pin_list->pin);
 		data->entry.trigger = entry.trigger;
 		data->entry.polarity = entry.polarity;
-		pr_debug("irq[%d] update trigger[%d] polarity[%d]\n",
-			irq, entry.trigger, entry.polarity);
 	}
 }
 EXPORT_SYMBOL(ioapic_sync_hardware_data);
-- 
2.17.1

